### ISSUE TITLE
[Feature Request] Cycle workflow with keyboard hotkey

### ISSUE THREAD

**Issue by rapatel0** [ebullient/obsidian-task-collector#66]
Feature description

In my normal workflow, i would like to be able to quickly cycle through a sequence of checkbox types to quickly assign status of a checkbox. 

As an example, I might want a workflow sequence with:
```
- [ ] (status not done /empty )
- [/] (In process / incomplete)
- [s] (scheduled}
- [w] (waiting) 
- [x] (done)
```

Hitting `ctrl-enter` four times would.
1. Convert the line to a checkbox (with empty status)
2. Switch to `[/]` Incomplete status
3. Switch to `[s]` scheduled status
4. Switch to `[w]` waiting status

Feature Configuration
- settings field with a configurable sequence of checkbox characters.
- configurable hotkey with `ctrl-enter` as default to advance sequence (go from empty to incomplete)
- configurable hotkey with `shift-enter` as default to reverse sequence (go from scheduled to incomplete)

**Comment by rapatel0** [comment]
Yeah, I know. I'm currently using that, but it's currently a bit clunky as a workflow. I want to enforce the progression from one status to another status. Going from in progress to waiting for example. 

It would be a lot easier and faster to just hold down the CTRL button and then hit enter 3x. And if one makes a mistake, just hit shift enter a few times.

**Comment by rapatel0** [comment]
This was implemented as another plugin https://github.com/thingnotok/obsidian-toggle-list

@ebullient  I think we can close this

**Comment by ebullient** [comment]
I see you forked the plugin to support this. 

Since you have it working, should I add it back? I confess it was a little strange to see my modal in a different plugin.. :-P

**Comment by ebullient** [comment]
This is included in the 1.0 release (BRAT at the moment)
